.github/workflows-source/ci-main.yml

```
@@ -21,6 +21,7 @@ permissions: read-all
 
 env:
   CI_COMMIT_SHA: ${{ github.sha }}
+  CI_COMMIT_BRANCH: ${{ github.head_ref || github.ref_name }}
```

Shouldn't that be the other way around like:
```suggestion
  CI_COMMIT_BRANCH: ${{ github.ref_name || github.head_ref }}
```

As in, we first want to use the name of the branch and if there's no name use the commit?

---

`github.head_ref`:
> The head_ref or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is either pull_request or pull_request_target.

`github_ref_name`:
>The short ref name of the branch or tag that triggered the workflow run. This value matches the branch or tag name shown on GitHub. For example, feature-branch-1. For pull requests, the format is <pr_number>/merge.

---

@marko-k0 then probably worth leaving a comment!